File: Domain/Stock.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain
{
    public class Stock
    {
        public Articulo articulo { get; set; }
        public Talle talle { get; set; }
        public Color color { get; set; }
        public int stock { get; set; }
    }
}

File: Grupo7/src/main/java/org/example/domain/Stock.java
package org.example.domain;

public class Stock {
    private int codigo;
    private String color;
    private String talle;
    public int stock;

    public Stock(int codigo, String color, String talle, int stock) {
        this.codigo = codigo;
        this.color = color;
        this.talle = talle;
        this.stock = stock;
    }

    public boolean tieneCodigo(int codigo) {
        return this.codigo == codigo;
    }

    public boolean hayStock() {
        return this.stock > 0;
    }
}
